UI/tmpl/index.html

```diff
@@ -38,6 +38,7 @@
     <a href="manual/index.html" target="_blank" class="header-link">K&auml;ytt&ouml;ohje</a>
     <a class="header-link" id="userNotification" style="cursor: pointer;">Tiedotteet</a>
     <a href="#work-list/municipality" class="header-link" title="Kuntasivun ohjeistus on vielä työn alla. Tarkempi ohjeistus lisätään käyttöohjeeseen viikolla 7.">Tietolajien kuntasivu</a>
+    <a href="#work-list/createdLinearAssets" class="header-link added-linear-assets" title="Lisätyt viivamaiset kohteet">Lisätyt viivamaiset kohteet</a>
```

Comment:
Minor check, but just see if the title attribute should be the same as the element text.

Reply:
> Minor check, but just see if the title attribute should be the same as the element text.

agreed, doesn't really make sense

Reply:
the title of this will be modified to "Lista viivamaisista kohteista, jotka luodaan yöllä liikennemerkkien luomisen/päivityksen seurauksena"